app/src/main/kotlin/codes/chrishorner/socketweather/home/HomeViewModel.kt

```diff
@@ -42,20 +46,28 @@ class HomeViewModel(
   private val timeFormatter = DateTimeFormatter.ofPattern("h a")
   private val scope = overrideScope ?: viewModelScope
 
-  val states: StateFlow<HomeState> = tickerFlow(10_000, emitImmediately = true)
-    .flatMapLatest { forecaster.states }
-    .combine(locationStore.savedSelections) { forecasterState, savedSelections ->
-      forecasterState.toHomeState(savedSelections)
-    }
+  private val showExtendedDescription: MutableStateFlow<Boolean> = MutableStateFlow(false)
```

Comment:
I'm looking to overhaul how viewmodels work in a bit. For now, I reckon for now this would be simpler if the expanded state was kept in the composable with a
```kotlin
var expanded by remember { mutableStateOf(false) }
```

That way we don't have to futz around with the viewmodel. It can still determine whether or not is `hasExtended`, but whether or not it's currently extended can be left as "view state" for now. (Though will likely change with some ideas I have in the future)

Reply:
That's actually my preference and I did it this way originally but using `rememberSaveable` because otherwise it forgets the expanded state on rotation; however, `rememberSaveable` doesn't appear to work (I couldn't figure out if I was doing something wrong but I did notice that the visible state of the location chooser uses it too and also doesn't survive rotation) so I opted for the VM route. Should I just use `remember` for now?

Reply:
Yeah, I reckon we stick with remember for now 👍